src/main/webapp/lectures.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* Used to gather URL parameters. */
+const URL_PARAM_ID = 'id';
+
+const REDIRECT_PAGE = '/lecture-view.html';
+
+loadLectureList();
+
+/**
+ * Fetches data from servlet and sets it in the lecture selection portion.
+ * Called whenever lecture selection page is loaded.
```

Comment:
No need to mention when the method is called. You only need to mention what the method does.